druid/src/command.rs

```diff
@@ -473,8 +507,8 @@ impl Command {
         self.get(selector).unwrap_or_else(|| {
             panic!(
                 "Expected selector \"{}\" but the command was \"{}\".",
-                selector.symbol(),
-                self.symbol
+                selector.symbol().str,
+                self.symbol.str
```

Comment:
instead of these changes I would manually implement `Display` for `SelectorSymbol`. Something like,


```rust
fn fmt(&self, f: Formatter) -> Result<_, _> {
    let must_use = if self.must_use {" (must use" } else { "" };
write!(f, "{}{}", self.str, must_use)
}
```

Reply:
I manually implemented `Debug` now. This output is not meant for end user